src/components/MaterialUI/Inputs/Radio.js

```diff
@@ -0,0 +1,113 @@
+import React, { useContext } from "react";
+import clsx from "clsx";
+import RadioGroupMui from "@material-ui/core/RadioGroup";
+import RadioMui from "@material-ui/core/Radio";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import FormControl from "@material-ui/core/FormControl";
+import FormLabel from "@material-ui/core/FormLabel";
+import { IntlContext } from "react-intl";
+import { makeStyles } from "@material-ui/core/styles";
+import RadioProps from "./RadioProps.js";
+
+const useStyles = makeStyles(theme => ({
+	/* Radio Button */
+	radioIcon: {
+		border: "1px solid",
+		borderColor: theme.palette.grey.borders,
+		width: theme.spacing(1.6),
+		height: theme.spacing(1.6),
+		borderRadius: "50%",
+		".MuiRadio-root:focus &, .MuiRadio-root:active &": {
+			boxShadow: `0 0 4px ${theme.palette.focus}`,
+			outline: "none",
+		},
+	},
+	radioIconChecked: {
+		borderColor: "currentColor",
+		"&:before": {
+			content: `" "`,
+			backgroundColor: "currentColor",
+			width: theme.spacing(1),
+			height: theme.spacing(1),
+			position: "absolute",
+			borderRadius: "50%",
+			top: "50%",
+			left: "50%",
+			transform: "translate(-50%, -50%)",
+		},
+	},
+}));
+
+const maybeTranslate = (formatMessage, message) => (message?.id ? formatMessage(message) : message);
```

Comment:
Do we have such pattern somewhere else? Should we not assume that all labels will already be localized like in the Checkbox or the Table components ? I would say it is the responsibility of the component user to make sure the label is set properly.

Reply:
In the OMS Toolbar, nothing was translated. Both the button and input are translated in orc-shared.
Input.js does the MaybeTranslate.
IconButton uses Text.js which does the translation.

Reply:
I would say that this is not the approach we take for Material-UI wrapper components we create. We want to eventually deprecate Gert's components you are referring to. I understand you point though but we should keep the same recipe for all of our wrappers. I am not saying the current one is the best one, I am just saying we should keep consistency throughout our code when related to Material-UI. Even Gert's toolbar might be replaced eventually.

Reply:
Alright, so you'd rather Radio doesn't translate and i do the translation in OMS useConfig where the tools are defined?

Reply:
I would say yes, especially that the label can also be a component or something else. I wouldn't give the responsibility of translation to our components.